src/stylus/generic/_transitions.styl

```diff
@@ -47,7 +47,7 @@
     opacity: 0
 
   &-enter-to
-    transtion: translate(0, 0)
+    transform: translate(0, 0)
```

Comment:
Its behavior happens implicitly, so just remove it

Reply:
Done.